install_requires.txt

```diff
@@ -1,4 +1,4 @@
 Django>=1.11,<3.0.0
 djangorestframework>=3.8.0,<3.12.0
-inflection==0.3.1
+inflection==0.4.0
```

Comment:
@simkimsia @suavesav  - As a retroactive point of feedback, requirements in `install_requires` should be as permissive as possible as to reduce conflicting requirements in the upstream app. I don't have context on how strict this requirement should be but should probably be `>=0.4,<0.5` to at least allow for the entire 0.4.x range, or even `>=0.3,<=0.5` unless there are any known issues with those versions. (Also, to be clear, I understand it was already pinned to a specific minor version, so not assigning blame here.)

Reply:
Copy that do u need me to change? And also I like to expand the range. Cause now there's 0.5.1